pom.xml

```diff
@@ -18,6 +18,7 @@
         <maven.checkstyle.plugin.configLocation>checkstyle.xml</maven.checkstyle.plugin.configLocation>
         <lombok.mapstruct.binding.version>0.2.0</lombok.mapstruct.binding.version>
         <mapstruct.version>1.5.5.Final</mapstruct.version>
+        <liquibase.version>4.22.0</liquibase.version>
```

Comment:
I think we can not add this property, because liquibase there is in spring boot starter, we can use reference to version